### DISCUSSION TITLE
Avoiding ES2021 in the client?

### DISCUSSION THREAD

**Discussion by bkrausz** [markdoc/markdoc#349] [Q&A] [answered]
We're running into the issue that Markdoc (via the NextJS plugin) is distributed as ES2021, and our site is targeting ES5. We tried using NextJS's `transpilePackages` on `@markdoc/markdoc`, but are seeing an `exports not defined` error.

Any advice on gaining broader browser support would be appreciated...my inclination is the simplest path would be to fork both @markdoc/next.js and @markdoc/markdoc to change the target, which doesn't sound particularly simple.

Am I missing something? I'm guessing this is the result of the NextJS plugin being in beta? Anyone have ideas on how to get it to not be bundled into the client (I've tried both static and server, it still seems to be bundled).

**Comment by mfix-stripe** [comment, marked as answer]
Get @bkrausz, not sure what is going on with `transpilePackages` (maybe you just need to upgrade to `next@13.1`?), but maybe the error is coming from `@markdoc/next.js` instead? Or maybe you can adjust your Webpack config to transpile `@markdoc/markdoc` 

There is a bunch of options for this, but here are a couple I found:
1. https://www.npmjs.com/package/next-transpile-modules
2. Example of adding a new rule to the Webpack config: https://github.com/carbon-app/carbon/blob/main/next.config.js#L15-L19
